Examples/Tests/implicit/CMakeLists.txt

```
@@ -31,6 +31,18 @@ add_warpx_test(
     OFF  # dependency
 )
 
+if(WarpX_PETSC)
```

Wait, is there even a `WarpX_PETSC` CMake option yet?

---

This should work for now:
```suggestion
if(AMReX_PETSC)
```

Do we use PETSC in WarpX at all or just _through_ AMReX?